kubernetes/debug.yml

```diff
@@ -47,6 +47,8 @@ spec:
         cpu: '0.5'
         memory: '300Mi'
     command: ["sleep", "9999"]
```

Comment:
`cat -v` also works and doesn't have a timeout.

Reply:
I'd rather have the timeout so it eventually dies ... also `sleep` is more readable